include/bluetooth/gap.h

```diff
@@ -69,6 +69,44 @@ extern "C" {
 #define BT_GAP_INIT_CONN_INT_MIN                0x0018  /* 30 ms    */
 #define BT_GAP_INIT_CONN_INT_MAX                0x0028  /* 50 ms    */
 
+/* LE PHY types */
+#define BT_GAP_LE_PHY_1M                        0x01 /* LE 1M PHY */
+#define BT_GAP_LE_PHY_2M                        0x02 /* LE 2M PHY */
+#define BT_GAP_LE_PHY_CODED                     0x03 /* LE Coded PHY */
```

Comment:
@rugeGerritsen Setting these as BITs instead would perhaps be a better idea? And convert the HCI LE Advertising Report to value to this define in the `bt_le_scan_recv_info` struct.

Reply:
@joerchan . Yes you are right. That provides more flexibility for future additions. You may consider adding `BT_GAP_LE_PHY_CODED_S2` and `BT_GAP_LE_PHY_CODED_S8`. It is probably cleaner to keep that out and use separate APIs for that.